### PR TITLE
feat: output password

### PR DESCRIPTION
**Pull request by aldenquimby**
## Background
- My team uses [Flyway](https://flywaydb.org/) for database migrations
- Flyway uses the JDBC postgres driver, which doesn't work with the output `db_url`
   - instead of `postgres://USER:PASS@HOST/DB`, it's `jdbc:postgresql://HOST/DB?user=USER&password=PASS`

## Proposal
- This PR adds `password` as an output, so that we can run Flyway migrations (or any other non-psql based migration system) after `create-branch-action`

## Alterantives
- I could instead store a separate `secrets.NEON_PASSWORD`, but I'd prefer to only need `secrets.NEON_API_KEY` given that `neonctl` returns everything I need
- I could use `neonctl` directly instead of this action

### CODE REVIEW

**Comment by aldenquimby** [comment]
@duskpoet thoughts on this? My team is using a fork for now

**Review by duskpoet — APPROVED** [review]
I needed time to check this out and validate it's working. Thanks, it should be fine to merge!

**Comment by aldenquimby** [comment]
@duskpoet great, thank you! I don't have write access to the repo, so will leave it to you to merge. Appreciate the quick reviews

**Pull request merged**
